src/components/technologies/genericTechnologies/res.py

```diff
@@ -13,17 +13,22 @@
 
 class Res(Technology):
     """
-    Resembles a renewable technology with no input. The capacity factors of the technology are determined for each
-    individual technology type.
+    Renewable technology with capacity factor (has no input)
 
-    **Parameter declarations:**
+    Resembles a renewable technology with no input. The capacity factors of the
+    technology are determined for each individual technology type.
 
-    - Capacity Factor of technology for each time step.
+    So far the following renewable technologies are possible:
+
```

Comment:
i remember we have a function here for the ST, do we delete it? Also there were a lot of todos and fixme, can they be deleted as well?

Reply:
The ST coding is an issue, as it is not done yet. I also deleted the todos that can be deleted and added the rest as issues

Reply:
perfect thanks